ui/src/components/tracks/dataset_slice_track.ts

```diff
@@ -381,6 +381,9 @@ export function renderTooltip<T>(
   return [
     m('', exists(durationFormatted) && m('b', durationFormatted), ' ', title),
     extras,
+    slice.count > 1
```

Comment:
You can use short circuit instead of a ternary to make this a little shorter.

```ts
slice.count > 1 && m('div', `and ${slice.count - 1} other events`)
```

here and below